Ingestion/fargate-login.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+set -x
```

Comment:
So this gets us into the ECS container?

Reply:
use sample ./fargate-login.sh to login to container, for example "./fargate-login.sh dev-ingestion"